matlab/test/arrow/tabular/tTabularDisplay.m

```diff
@@ -124,6 +153,167 @@ function ManyRowsAndColumns(testCase, TabularType)
+            var3Field = char(makeFieldString("Var3", "Time64", "arrow.type.Time64Type"));
+            expectedDisplay = [char(header), newline, ...
+                               newline, ...
+                               '    Schema:', newline, ...
+                               newline, ...
+                               '        ', var1Field, ' | ', var2Field, ' | ', var3Field, newline, ...
+                               newline, ...
+                               '    First Row:', newline, ...
+                               newline, ...
+                               '        ', '2023-01-01 00:00:00.000000 | <Struct> | 00:00:01.000000', newline,...
+                               newline];
+
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+        end
+
+        function displayInvalidData(testCase, TabularType)
+            % Display "null" for invalid value
+            import arrow.internal.test.display.makeLinkString
+
+            t = table(seconds([NaN;1]), string([missing;"a"]), string(["";"b"]),  [NaN;1], datetime(2023,1,[NaN;2]),...
+                      VariableNames=["durationVar", "stringVar1", "stringVar2", "doubleVar", "datetimeVar"]);
+            arrowTabularObj = TabularType.FromTableFcn(t);
+            actualDisplay = evalc('disp(arrowTabularObj)');
+
+            classNameString = makeLinkString(FullClassName=TabularType.FullClassName, ...
+                ClassName=TabularType.ClassName, BoldFont=true);
+            header = compose("  Arrow %s with %s rows and %s columns:", classNameString, getNumString(2), getNumString(5));
+            var1Field = char(makeFieldString("durationVar", "Time64" , "arrow.type.Time64Type"));
+            var2Field = char(makeFieldString("stringVar1", "String" , "arrow.type.StringType"));
+            var3Field = char(makeFieldString("stringVar2", "String", "arrow.type.StringType"));
+            var4Field = char(makeFieldString("doubleVar", "Float64", "arrow.type.Float64Type"));
+            var5Field = char(makeFieldString("datetimeVar", "Timestamp", "arrow.type.TimestampType"));
+            expectedDisplay = [char(header), newline, ...
+                               newline, ...
+                               '    Schema:', newline, ...
+                               newline, ...
+                               '        ', var1Field, ' | ', var2Field, ' | ', var3Field, ' | ', var4Field, ' | ', var5Field, newline, ...
+                               newline, ...
+                               '    First Row:', newline, ...
+                               newline, ...
+                               '        ', 'null | null | "" | null | null', newline,...
+                               newline];
+
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+        end
+
+        function unicodeDisplayWell(testCase, TabularType)
+            % The current display doesn't align anything. So it should work
```

Comment:
This is a bit confusing. I would say the current display does not attempt to align multiple rows vertically. Therefore, it should work well displaying Unicode characters.